docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>Also, the first line is too hard to follow since you're performing a factorization AND a multiplication at the same time. Can you please split up those two steps?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Yes. So, I can first keep the fraction format and just distribute the <code>\sum</code> into the numerators of fractions, and THEN, provide the rest of equations. I will add this step before the first line.</p>